### ISSUE TITLE
Label is not provided for the edit field under the filter section in Windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-react#3580]
### Description

**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7670)
Browsers: Edge (Version 145.0.3800.22), Chrome (Version 144.0.7559.110)
Screen readers: NVDA (2025.3.2), JAWS (2026.2512.120), Narrator
URL: https://www.telerik.com/kendo-react-ui/components/grid

**User Impact:**
Screen reader users may not understand the purpose of the edit field because no label is provided. The screen reader may only announce a generic “edit field,” causing confusion about what information is required. This can prevent users from completing filtering tasks independently and increases cognitive effort.

### Plan

KendoReact Free

### Steps To Reproduce

**Pre-Requisite:**

Turn on Screen Readers:

- NVDA: Ctrl + Alt + N
- JAWS: Ctrl + Alt + J
- Narrator: Ctrl + win + Enter

Verbosity:

- NVDA: Default
- JAWS: Beginner, Highest
- Narrator: Default

**Repro Steps:**

1. Open the URL: https://www.telerik.com/kendo-react-ui/components/grid in edge browser in windows.
2. 'React Data Grid Overview' page will open.
3. Press tab key to reach to the 'More Option' button and activate it.
4. Press tab key to reach to the 'Filters' control and activate it.
5. Filters controls will appear.
6. Press tab key to navigate to the edit field present under Filter control.
7. Verify whether label is provided for the edit field under the filter section in Windows or not.

### Screenshots or video

**Refer Attachment:**

1. Label is not provided for the edit field under the filter section in Windows.png
2. Label is not provided for the edit field under the filter section in Windows.mp4

https://github.com/user-attachments/assets/1c92d50e-f4b8-4c18-9ecd-4437a2d2dc11
<img width="975" height="528" alt="Image" src="https://github.com/user-attachments/assets/6f927564-6254-4821-8507-e3a29962aacd" />

### Actual Behavior

Label is not provided for the edit field under the filter section in Windows.
Screen reader announces: edit blank

### Expected Behavior

Label should be provided for the edit field under the filter section in Windows.
Screen Reader should announce: Value edit blank

### Browser

Chrome, Microsoft Edge

### Browser version

Edge (Version 145.0.3800.22), Chrome (Version 144.0.7559.110)

### OS type

Windows

### OS version

Windows 11 version 25H2 (OS Build 26220.7670)

**Comment by PekoPPT** [comment]
HI, @msftedad. 
Thank you for the shared details. I am marking the reported behavior as a bug.

**Comment by msftedad** [comment]
Hi @PekoPPT, Any update on this issue?
